src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperComponent.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Microsoft.Extensions.Internal;
+
+namespace Microsoft.AspNetCore.Razor.Runtime.TagHelpers
+{
+    public class TagHelperComponent : ITagHelperComponent
+    {
+        /// <inheritdoc />
+        /// <remarks>Default order is <c>0</c>.</remarks>
+        public virtual int Order => 0;
+
+        /// <inheritdoc />
+        public virtual void Init(TagHelperContext context)
+        {
+        }
+
```

Comment:
Probably makes sense to add a helper `Process` method here like we have on `TagHelper`